styles.css

```
@@ -115,4 +115,27 @@
         border-radius: 20px;
         border: 0;
     }
+
+    footer{
+        height: 132px;
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        flex-wrap: wrap;
+    }
+
+    .social{
+        display: flex;
+        justify-content: space-between  ;
+        align-content: center;
+        width: 150px;
```

yo no dejaría el valor fijo, sino mas bien maximo

```suggestion
        max-width: 150px;
```